activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Prevent AbstractAdapter from converting exceptions from ActiveSupport::Notification
+    callbacks into ActiveRecord::StatementInvalids.
```

Comment:
There's no need to mention `AbstractAdapter` here, it's an implementation detail.

We should include the name of the notification event that was affected (`sql.active_record`).

References to constants, methods or options should be wrapped in backticks.

Putting it all together 💅:

> Prevent errors raised by `sql.active_record` notification subscribers from being converted into `ActiveRecord::StatementInvalid` exceptions.